packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -103,6 +103,15 @@ export class Dialog
    */
   @Prop({ mutable: true }) embedded = false;
 
+  /**
+   * When `true`, disables the default close on escape behavior.
+   *
+   * By default, an open dialog can be dismissed by pressing the Esc key. Depending on what the dialog represents, it may not be desired for this behavior.
```

Comment:
Nitpick: "... represents, this behavior may not be desired."

Reply:
I think the second sentence here could be removed.